flocker/node/agents/functional/test_gce.py

```diff
@@ -0,0 +1,97 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Functional tests for ``flocker.node.agents.gce`` using a GCE cluster.
+
+In order to run these tests you'll need to define the following environment
+variables:
+
+``FLOCKER_FUNCTIONAL_TEST=TRUE
+FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_FILE=$HOME/acceptance.yml
+FLOCKER_FUNCTIONAL_TEST_CLOUD_PROVIDER=gce``
+
+The configuration stanza for the GCE backend should resemble:
+``
+gce:
+    zone: <gce-region>
+    project: <gce-project-name>
+``
```

Comment:
:question: It doesn't matter because we don't render docs, but block code syntax is:

```rst

blah blah the following environment variables::

    FLOCKER_THING=aoeusa
    FLOCKER_Ouoa

```

(two colons, blank line, four space indent, blank line)